config/default.py

```diff
@@ -16,6 +16,7 @@
 SECURITY_PASSWORD_HASH = "bcrypt"
 SECURITY_PASSWORD_SALT = "abcde1234509876zyxwvu22"
 SECURITY_LOGIN_USER_TEMPLATE = "index.html"
+SECURITY_REGISTER_USER_TEMPLATE = "index.html"
```

Comment:
All this does is get rid of "/register" as a page you can go to by making it just be the index page. Same exact thing we did to login.